docs/en/wallet-attestation.rst

```diff
@@ -10,33 +10,32 @@ Wallet Attestation contains information regarding the security level of the devi
 Requirements
 ------------
 
-The following requirements for the Wallet Attestation are met:
+The following requirements for the Wallet Attestation must be met:
```

Comment:
since we use here must and then within each requirement other language like should or may, I suggest to reformulate the sentence

```suggestion
The requirements for the Wallet Attestation are defined below:
```